components/gitpod-protocol/src/usage.ts

```diff
@@ -42,7 +42,6 @@ export interface ExtendedBillableSession extends BillableSession {
 
 export interface BillableSessionRequest {
     attributionId: string;
-    startedTimeOrder: SortOrder;
```

Comment:
Should we not keep ordering information in the protocol especially if we do server-side pagination?

Reply:
given that this feature is currently not working at all I wanted to start with tidy first.